django/contrib/staticfiles/management/commands/collectstatic.py

```diff
@@ -145,6 +145,7 @@ def collect(self):
         }
 
     def handle(self, **options):
+        self.check(tags=['staticfiles'])
```

Comment:
You'll want to support `--skip-checks` if you add a check. Since the option only gets added when `requires_system_checks = True` you'll need to add it to `add_arguments` see https://github.com/django/django/commit/0b83c8cc4db95812f1e15ca19d78614e94cf38dd#diff-e835ddfb52774b39749788a0d046e477.

This makes me wonder if we want to replace `Command.requires_system_checks = True` by something like `Command.required_system_checks = '__all__'` that can be set to an empty list to achieve `required_system_checks = False` and allow subclasses that want only a subset to specify tags through it e.g. `required_system_checks = ['staticfiles']`. That would prevent having do to the manual and error prone `options['skip_checks']` dance.

Reply:
Thanks @charettes . I added the `--skip-checks` option as you mentioned.
Also, I created a [ticket](https://code.djangoproject.com/ticket/31546) based on you suggestion for `Command.requires_system_checks`